src/types/index.ts

```diff
@@ -337,7 +337,7 @@ export interface ClaimScope {
 }
 
 export interface TrustedClaimIssuer {
-  identity: Identity;
+  identity: string | Identity;
```

Comment:
The way we handle these situations in the SDK is by creating a specific `Input` type (`InputTrustedClaimIssuer` or `TrustedClaimIssuerLike`)

Something like 
```
export type InputTrustedClaimIssuer = Modify<TrustedClaimIssuer, {
  identity: string | Identity;
}>;
```

The name is up to you

We can then use that type where users are required to input said values, without affectinig the return types of getters

This also applies to `IdentityCondition`, but leave that one as it was before because I'm working on it on my end and we'll have some pretty awful conflicts otherwise

Reply:
Thanks, this looks a lot better know.